plugins/email_service/app/helpers/email_service/application_helper.rb

```diff
@@ -1006,7 +1293,7 @@ def delete_custom_verification_email_template(name)
         status = "#{e.message}"
         Rails.logger.error "CRONUS: DEBUG: Create custom verification template: #{e.message}"
```

Comment:
you have debug as string in the log error

Reply:
removed.